internal-api/src/main/java/datadog/trace/api/iast/sink/UntrustedDeserializationModule.java

```diff
@@ -2,9 +2,14 @@
 
 import datadog.trace.api.iast.IastModule;
 import java.io.InputStream;
+import java.io.Reader;
 import javax.annotation.Nullable;
 
 public interface UntrustedDeserializationModule extends IastModule {
 
   void onInputStream(@Nullable InputStream is);
```

Comment:
Related with my other comments

```suggestion
  void onObject(@Nullable Object obj); 
```